sdks/go/client/multiple_nodes_client.go

```diff
@@ -36,13 +36,17 @@ func NewMultipleNodesClient(urls []string) (*MultipleNodesClient, error) {
 }
 
 func (c *MultipleNodesClient) FetchLatestBlockHeight(ctx context.Context) (uint64, error) {
+	var errs []error
 	for _, node := range c.nodes {
 		height, err := node.FetchLatestBlockHeight(ctx)
 		if err == nil {
 			return height, nil
+		} else {
+			fmt.Printf("Encountered an error while attempting to fetch latest block height from one node. Error: %v\nNode: %v\n", err, node)
```

Comment:
would we be even able to see this printf, I am not sure

Reply:
I think we should be able to see them, but I'm not 100% sure. I think it makes the most sense to use some standard library logging functionality rather than the go-ethereum package if we are doing the logging in the client itself. 

That's why my ultimate goal would be to alter the function signature to return the intermediate errors in some array that doesn't represent the function call failing so that the caller can log them however they choose to.